katharsis-jpa/src/main/java/io/katharsis/jpa/internal/JpaRepositoryBase.java

```diff
@@ -50,6 +55,28 @@
 		}
 	}
 
+	
+	/**
+	 * By default LookupIncludeBehavior.ALWAYS is in place and we let the relationship repositories load the relations. There 
+	 * is no need to do join fetches, which can lead to problems with paging (evaluated in memory instead of the db).
+	 * 
+	 * @param fieldName
+	 */
+	protected boolean fetchRelations(String fieldName) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused method parameter "fieldName". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1172)